src/site/pages/contribution.md

```diff
@@ -41,6 +41,7 @@ There are several properties available on the Example component to help with sty
 <Example heading="Heading" minHeight="20em" docs={docReference} fullPageOnly="">{example}</Example>;
 ```
 
+If an ID is needed for an element, please use the Handlebars unique ID helper.  This can be used by appending {{uniqueId} to the ID attribute so it looks like this:  id="badge-{{uniqueId}}".
```

Comment:
Can you backtick the code bits so they look like code? 
Also, `{{uniqueID}` needs one more `}`
Thanks for remembering to add it to the docs!